server/checks/Approval.js

```diff
@@ -231,28 +229,56 @@ export default class Approval extends Check {
   }
 
   /**
+   * Gets the users to ignore for a pull request in a repository, according to its
+   * approval configuration.
+   *
+   * @param github The GithubService instance
+   * @param repository The repository
+   * @param pull_request The pull request
+   * @param config The approval configuration
+   * @param token The access token to use
+   * @returns {Array} The logins of users to ignore
+   */
+  static async fetchIgnoredUsers(github, repository, pull_request, config, token) {
+    if (!config.ignore || config.ignore === 'none') {
+      return []
+    }
+    const ignoreConfig = config.ignore
+    const user = repository.owner.login
+    const repoName = repository.name
+    const ignoredUsers = []
+    if (ignoreConfig === 'last_committer' || ignoreConfig === 'both') {
+      const lastCommitter = await github.fetchLastCommitter(user, repoName, pull_request.number, token)
+      if (lastCommitter) {
+        ignoredUsers.push(lastCommitter)
+      }
+    }
+    if (ignoreConfig === 'pr_opener' || ignoreConfig === 'both') {
```

Comment:
Then `both` would not work correctly, right? It' execute the block of the first `if` and skip the second because of the `else`.